src/main/java/com/vertexvis/example/JobPoller.java

```diff
@@ -1,54 +0,0 @@
-package com.vertexvis.example;
-
-import com.vertexvis.model.QueuedJob;
-
```

Comment:
Why are all of these examples removed. IIRC, these were added so clients could reference these as to how to use the platform with the java client.